src/core/zowe/core_for_zowe_sdk/request_handler.py

```diff
@@ -72,6 +72,30 @@ def perform_request(self, method, request_arguments, expected_code=[200]):
         self.__validate_response()
         return self.__normalize_response()
 
+    def perform_streamed_request(self, method, request_arguments, expected_code=[200]):
+        """Execute a streamed HTTP/HTTPS requests from given arguments and return a raw response.
+
+        Parameters
+        ----------
+        method: str
+            The request method that should be used
+        request_arguments: dict
+            The dictionary containing the required arguments for the execution of the request
+        expected_code: int
+            The list containing the acceptable response codes (default is [200])
+
+        Returns
+        -------
+        A raw response data
+        """
+        self.method = method
+        self.request_arguments = request_arguments
+        self.expected_code = expected_code
+        self.__validate_method()
+        self.__send_stream_request()
```

Comment:
Would it make sense to add an optional `stream=True` parameter on the existing `__send_request` method?